packages/block-editor/src/layouts/flow.js

```diff
@@ -157,20 +157,62 @@ export default {
 		return 'vertical';
 	},
 	getAlignments( layout ) {
+		const alignmentInfo = getAlignmentsInfo( layout );
 		if ( layout.alignments !== undefined ) {
```

Comment:
Is this something we can deprecate maybe? (the alignment property in the flow layout, I think I added this initially before having `content` and `wide` sizes. It might not be useful anymore but I'm not sure.

Reply:
Currently we don't have a way to set this with a control, but if we set it in `code editor` it works. I haven't checked if provided in `theme.json`, but I guess this is where it could restrict some alignments..

If the `theme.json` use case doesn't make sense, I guess it's okay to remove. I'll have to check this better though..

Reply:
I think the use-cases don't make sense, it's just a historic thing now.

Reply:
I see that it's still used in some blocks (that should be refactored to flex layout): buttons, navigation.

Reply:
We should revisit with the refactoring of these blocks. Sounds good?

Reply:
Yep, makes sense. Thank you.